src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -588,18 +384,18 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 			suggestion = this._suggestWidget?.getFocusedItem();
 		}
 		const initialPromptInputState = this._mostRecentPromptInputState;
-		if (!suggestion || !initialPromptInputState || !this._leadingLineContent || !this._model) {
+		if (!suggestion || !initialPromptInputState || this._leadingLineContent === undefined || !this._model) {
 			return;
 		}
-		this._lastAcceptedCompletionTimestamp = Date.now();
+		SuggestAddon.lastAcceptedCompletionTimestamp = Date.now();
```

Comment:
We don't want this to be static as it's intentionally tracked per suggest addon. It's unlikely to cause a problem if it were static, but it makes the code a bit harder to understand being static imo.

Reply:
this is referenced in the pwshAddon https://github.com/microsoft/vscode/blob/e5bfb5483f8420fa57d27af9ded84bc818efa29f/src/vs/workbench/contrib/terminalContrib/suggest/browser/pwshCompletionProviderAddon.ts#L261

Reply:
Still, there are many suggest addons (1 per terminal), so it just confuses things imo to make it static.